include/deal.II/fe/fe_dgq.h

```diff
@@ -435,8 +435,14 @@ class FE_DGQArbitraryNodes : public FE_DGQ<dim, spacedim>
 /**
  * Implementation of scalar, discontinuous tensor product elements based on
  * Legendre polynomials, described by the tensor product of the polynomial
- * space Polynomials::Legendre. As opposed to the basic FE_DGQ element, these
- * elements are not interpolatory and no support points are defined.
+ * space Polynomials::Legendre. The tensor product is achieved using
+ * TensorProductPolynomials and the ordering of shape functions, like in
+ * TensorProductPolynomials, is lexicographic. For instance, the ordering in 2d
+ * is $P_0(x)P_0(y),\ P_1(x)P_0(y),\ \ldots,\ P_n(x)P_0(y),\ P_0(x)P_1(y),\
+ * \ldots,\ P_n(x)P_1(y),\ \ldots,\ P_0(x)P_n(y),\ \ldots,\ P_n(x)P_n(y)$ when
```

Comment:
I think you actually meant to have the space there, right? As in `\ ` is a protected space for latex. If you want that, you probably want to put the `\ ` at the front of the next line to make sure the space isn't eaten by any scripts looking for trailing whitespace.

Reply:
Yes! That was what I intended to. However the newline character also does the job, so the space is redundant here. Thanks for noticing!

Reply:
If you don't mind, move the backslash to the beginning of the next line anyway. Backslashes have special meaning at the end of the line for the C++ preprocessor, and one is left wondering whether the backslash you have here is going to have strange side effects when compiling...

Reply:
Yes, sure. Fixed in the following commit.